test/e2e/suite/issuers/vault/certificate/approle.go

```diff
@@ -68,8 +76,16 @@ var _ = framework.CertManagerDescribe("Vault Certificate (AppRole)", func() {
 	var secretId string
 	var vaultInit *vaultaddon.VaultInitializer
 
+	var vaultSecretNamespace string
+
 	BeforeEach(func() {
 		By("Configuring the Vault server")
+		if issuerKind == cmapi.IssuerKind {
+			vaultSecretNamespace = f.Namespace.Name
+		} else {
+			vaultSecretNamespace = "kube-system"
```

Comment:
I am pretty sure this *isn't* the cluster resource namespace during e2e tests.. the default helm chart will set the cluster issuer namespace to the same namespace that the cert-manager pod runs in 🤔

Reply:
Yep, you're quite right. Will update that